src/app/archive/lib/processor.ml

```diff
@@ -1845,7 +1845,7 @@ module User_command = struct
             (Caqti_request.find typ Caqti_type.int
                (Mina_caqti.insert_into_cols ~returning:"id" ~table_name
                   ~tannot:(function
-                    | "typ" -> Some "user_command_type" | _ -> None )
+                    | "command_type" -> Some "user_command_type" | _ -> None )
```

Comment:
Is some renaming of `typ` to `command_type` happening behind the scenes?

Reply:
The current schema in `create_schema.sql` uses `command_type` in the `user_commands` table.